data/geolocation-!cn

```diff
@@ -1,8 +1,5 @@
 # This list contains domains that don't have access point in China mainland. This is opposite to geolocation-cn.
 
-# Anime, Comics & Games
-include:category-acg
```

Comment:
Not a good idea.
You deleted domains from the collection `geolocation-!cn` for no good reasons.